README.adoc

```diff
@@ -173,7 +149,8 @@ include::{common-includes}/mvnbuild.adoc[]
 
 When the server is running, point your browser to the
 `http://localhost:9080/inventory/systems/localhost` URL.
-You receive the system properties of your local JVM from the `inventory` service. Next, point your
+You receive all of the system properties in your local JVM.
```

Comment:
I don't think typically the term, receive, is used in such cases?

The distinction between the inventory service and system service is unclear still in this paragraph or section?

Reply:
Fixed the "receive". 

The distinction between the inventory and system services had been explained through out the guide, ie, in the intro, and in the "Try what you'll build" section with the URL used there. Not sure we want repeat it again in the paragraph/section.